src/EPR.PRN.Backend.API/Services/PrnService.cs

```diff
@@ -28,7 +28,7 @@ public async Task<List<PrnDto>> GetAllPrnByOrganisationId(Guid orgId)
             return (await _repository.GetAllPrnByOrganisationId(orgId)).Select(x => (PrnDto)x).ToList();
         }
 
-        public async Task UpdateStatus(Guid orgId, List<PrnUpdateStatusDto> prnUpdates)
+        public async Task UpdateStatus(Guid orgId, Guid userId, List<PrnUpdateStatusDto> prnUpdates)
```

Comment:
Is there a need to check prnUpdates does not contain the same id more than once, perhaps with different statuses?